### DISCUSSION TITLE
What should be the name of the project?

### DISCUSSION THREAD

**Discussion by Prakash4844** [Prakash4844/tit#8] [General]
We need to decide the name for this project.

Here:
Potential names for VCS:

My Preference:

```
ICE
FOX
PIT
YIT
KIL
```

Controversal names:

```
TIT
DIC
DIQ
FUK
SHT
CNT
SEX
```

Normal Name:
here are some meaningful words based on the provided short names:

```
1. VEX: Vertex
2. ZEN: Zenith
3. RYZ: Rhythm
4. QUO: Quorum
5. JIV: Jive
6. ZIL: Zillion
7. KAZ: Kazoo
8. WEX: Wavelength
9. PIZ: Pizzazz
10. FIZ: Fizzle
11. XAN: Xanadu
12. BEX: Bexar (a county in Texas)
13. LOX: Lox (a type of smoked salmon)
14. VIK: Viking
15. QIX: Quixote (as in Don Quixote)
16. MOZ: Mozart
17. ZYX: The reverse alphabet
18. HUV: Hover
19. VYX: Vexed
20. NYX: Nyx (Greek goddess of the night)
21. AIM
22. JOY
23. SKY
24. GEM
25. TUM
```

This discussion is for #3

**Comment by Prakash4844** [comment]
After discussing it among me and @sagargoswami2001

We have decided to name it `TIT`

Since git is acronym for Global information tracker.

Tit can be an acronym for Transdimensional information tracker.

Also, Tit is a beautiful bird.

And who doesn't like a controversial name. 😂 😂

**Comment by SakuraBlossomTree** [comment]
okay so the name is TIT 


What is the usage of this and the various documentation

Give me the brief things about the VNS 

I will make the docs for it

**Reply by Prakash4844** [reply]
> okay so the name is TIT

Actually we are still open to suggestion, do you have any suggestion?

> What is the usage of this and the various documentation.

This is a Version control system, a Command Line Tool Inspired by `git` and are trying to mimic `git` itself. nothing much is implemented yet except for `--version | -v` and `--help | -h`

> Give me the brief things about the VNS

Actually i didn't get what is `VNS` Do you mind explaining.

**Reply by SakuraBlossomTree** [reply]
VNS was a typo i was asking about VNC


Okay so This is great if you trying to mimic git 

Because I have basic knowledge on git

And once you have finalized the name then I will start the documentation

If you want a Suggestion ```NYX```  sounds good

**Comment by Prakash4844** [comment]
> If you want a Suggestion `NYX` sounds good

Nyx is already a command-line monitor for [Tor](https://www.torproject.org/). so we should avoid it, as we don't want it to clash with other CLI utility.

It's Great that you have experience with `git`.

and I suppose you mean `Version control` as `VNC`? if yes please refer it as VCS `Version Control System` or SCM `Source Control management` which are popular term for it. VNC is usually used to refer to `Virtual Network Computing`

**Reply by SakuraBlossomTree** [reply]
Sure, I have the basic README.md File ready for now

Should I commit it

**Reply by Prakash4844** [reply]
create a pull request so we can review it.

**Reply by SakuraBlossomTree** [reply]
I have create a PR

**Comment by sagargoswami2001** [comment]
Name is changed.
